fails with GNU make

because changing directory with `cd` is not "permanent":

```
~/projects/highlightjs$ make
Cloning into 'highlight'...
remote: Counting objects: 13948, done.

. . .

HEAD is now at 3e9cd85... Version 8.3
npm ERR! install Couldn't read dependencies
npm ERR! Error: ENOENT, open '/home/szelga/projects/highlightjs/package.json'

. . .
```

what `make` flavour/version do you build this project with?

--- a/Makefile
+++ b/Makefile
@@ -1,15 +1,18 @@
 VERSION=8.3
 
-default: highlight
-		@cp -f highlight/build/highlight.* .
-		@cp -f highlight/src/styles/* styles
-		@du -hs highlight.*
+default: highlight/build
+	@cp -f highlight/build/highlight.* .
+	@cp -f highlight/src/styles/* styles
+	@du -hs highlight.*
+
+highlight/build: highlight
+	@cd highlight && git fetch && git checkout $(VERSION)
+	@cd highlight && npm install && node tools/build.js
 
 highlight:
-		@git clone git://github.com/isagalaev/highlight.js.git $@
-		@cd highlight && git pull && git checkout $(VERSION)
-		@npm install
-		@node tools/build.js
-		@cd ..
+	@git clone git://github.com/isagalaev/highlight.js.git $@
+
+clean:
+	@rm -rf highlight/build
 
 .PHONY: default

